spec/controllers/listings_controller_spec.rb

```diff
@@ -16,13 +16,26 @@ def create_transaction_type(community_id, type, process_id, translations = [], c
 
     defaults = TransactionTypeCreator::DEFAULTS[type]
 
+    # Save name to TranslationService
+    translations_with_default = translations.concat([{ locale: "en", name: type }])
+    name_group = {
+      translations: translations_with_default.map { |translation|
+          { locale: translation[:locale],
+            translation: translation[:name]
+          }
+        }
+      }
+    created_translations = TranslationService::API::Api.translations.create(community_id, [name_group])
+    name_tr_key = created_translations[:data].map { |translation| translation[:translation_key] }.first
+
     opts = defaults.merge(
       {
         shipping_enabled: false,
         transaction_process_id: process_id,
-        name_tr_key: 'something.here',
+        name_tr_key: name_tr_key,
         action_button_tr_key: 'something.here',
```

Comment:
These tests do not check action_button_label at the moment - so I didn't start to populate those yet.